### PR TITLE
etcdctl:readme.md doc made uniform

### PR DESCRIPTION
**Pull request by sinsharat**
1. Modified the document to display the output as per earlier examples.
2. fixed one error of extra quat present for bash
3. updated examples with just etcdctl mentioned under examples to ./etcdctl

### CODE REVIEW

**Comment by heyitsanthony** [comment]
@sinsharat I think the `#`'s are there to distinguish input from output. That way people can copy and paste the full example into a bash shell and it'll work. Update to use `#`'s for the output lines and this should be good to merge. Thanks!

**Comment by sinsharat** [comment]
@heyitsanthony sure then will modify the above outputs along with the changes i did with #.
Thanks!

**Comment by sinsharat** [comment]
@heyitsanthony i have done the changes as per your suggestion. Let me know if anything else is required.
Thanks!

File: etcdctl/README.md [hunk 1]
```diff
@@ -1082,18 +1092,18 @@ etcdctl --user=root:123 user grant-role userA roleA
...
 
 ## Notes
 
 - JSON encoding for keys and values uses base64 since they are byte strings.
 
 
-[etcdrpc]: ../etcdserver/etcdserverpb/rpc.proto
-[storagerpc]: ../mvcc/mvccpb/kv.proto
-[member_list_rpc]: ../etcdserver/etcdserverpb/rpc.proto#L493-L497
+#[etcdrpc]: ../etcdserver/etcdserverpb/rpc.proto
```
Comment (heyitsanthony) [inline etcdctl/README.md]: `#`'s don't belong here?

File: etcdctl/README.md [hunk 2]
```diff
@@ -1063,8 +1073,8 @@ etcdctl --user=root:123 user passwd myuser
 #### Examples
 
 ``` bash
-etcdctl --user=root:123 user grant-role userA roleA
-# Role roleA is granted to user userA
```
Comment (heyitsanthony) [inline etcdctl/README.md]: have spaces between the `#` and the output? `#abc` looks more cluttered than `# abc`

**Comment by sinsharat** [comment]
@heyitsanthony sure will do the changes.

**Comment by heyitsanthony** [comment]
lgtm after fixing the spacing and removing the non-output #'s. Thanks!

**Comment by sinsharat** [comment]
@heyitsanthony i have done the changes. let me know if its fine now.
Thanks!

**Comment by heyitsanthony** [comment]
OK, please squash into a single commit and this can be merged. Thanks!

**Comment by sinsharat** [comment]
@heyitsanthony i have squashed the commits.
Thanks!

File: etcdctl/README.md [hunk 3]
```diff
@@ -36,8 +36,8 @@ The protobuf encoding of the PUT [RPC response][etcdrpc].
 ./etcdctl put foo bar --lease=1234abcd
 OK
```
Comment (heyitsanthony) [inline etcdctl/README.md]: `# OK` (sorry, only caught this one on the final pass)

Comment (sinsharat) [inline etcdctl/README.md]: @heyitsanthony no issues will modify it.

**Comment by sinsharat** [comment]
@heyitsanthony Have done the changes.
Thanks!

**Pull request merged**